### PR TITLE
Awsbatch profiles

### PR DESCRIPTION
**Pull request by apeltzer**
This will add 

- Profile for AWSBatch
- Remove old AWS Profile (that probably nobody will use after AWSBatch works ?)
- Add some basic documentation for AWSBatch specific parameters

Furthermore, I started writing a function to check for allowed paths https://github.com/nextflow-io/nextflow/issues/813 as also discussed in issue https://github.com/nf-core/tools/issues/79 . 

Will test this later :-)

### CODE REVIEW

**Comment by apeltzer** [comment]
This should fix #51 once we're done with it :-)

**Comment by apeltzer** [comment]
Question: Should we remove AWS in general? I mean most people will probably use AWSBatch or fire up an EC2 instance and then use the  `base,docker` or `base,singularity` profiles ?

**Comment by ewels** [comment]
Yes, I think so. Those profiles were mostly back from when the default was for uppmax I think. From memory I don’t think that they really contained much.. (scheduler ignite perhaps?). But I think that batch should be the default AWS option for basically everyone.

**Comment by apeltzer** [comment]
I will remote the AWS configs then, they just confuse especially newbies then..

File: {{cookiecutter.pipeline_slug}}/main.nf [hunk 1]
```diff
@@ -26,7 +26,7 @@ def helpMessage() {
     Mandatory arguments:
       --reads                       Path to input data (must be surrounded with quotes)
       --genome                      Name of iGenomes reference
-      -profile                      Hardware config to use. docker / aws
+      -profile                      Hardware config to use. docker / awsbatch
```
Comment (ewels) [inline {{cookiecutter.pipeline_slug}}/main.nf]: This message is a bit misleading. Should probably be "Configuration profile to use.", and list _all_ of the available options.

**Review by ewels — DISMISSED** [review]
Looking really good! 👍

**Pull request merged**